libs/auth-api-lib/src/lib/services/delegations.service.ts

```diff
@@ -82,24 +87,29 @@ export class DelegationsService {
           { validTo: { [Op.or]: [{ [Op.eq]: null }, { [Op.gt]: now }] } },
         ],
       },
+      include: [DelegationScope],
     })
 
     return result.map(
       (d) =>
```

Comment:
You can just return `result.map(d => d.toDTO())`

Reply:
Good catch, forgot this one.